### PR TITLE
Webots R2020b blog post

### PR DESCRIPTION
**Pull request by omichel**
This is the [blog page](https://cyberbotics.com/doc/blog/Webots-2020-b-release?version=feature-R2020b-blob-post) announcing the release of Webots R2020b 🎉

### CODE REVIEW

**Comment by omichel** [comment]
This PR is ready for reviews and contributions.
Feel free to commit your contributions directly.

File: docs/blog/Webots-2020-b-release.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+However, this article only lists some of the major changes.
+Please refer to the [2020 changelog](../reference/changelog-r2020.md) for a comprehensive list of all changes.
+
+---
+
+## 4 New Robot Models
+
+### Spot from Boston Dynamics
+
+Play around with a simulated Spot robot from [Boston Dynamics](http://bostondynamics.com) in Webots.
```
Comment (DavidMansolino) [inline docs/blog/Webots-2020-b-release.md:18]:
```suggestion
Play around with a simulated [Spot robot](../guide/spot.md) from [Boston Dynamics](http://bostondynamics.com) in Webots.
```

File: docs/blog/Webots-2020-b-release.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+%figure "Spot Model in Webots"
+![Spot Model in Webots](images/spot.jpg)
+%end
+
+### P-Rob 3 from F&P Robotics  
+
+P-Rob 3 is a new collaborative robotic arm developed by [F&P Robotics](https://www.fp-robotics.com).
```
Comment (DavidMansolino) [inline docs/blog/Webots-2020-b-release.md:26]:
```suggestion
[P-Rob 3](../guide/p-rob3.md) is a new collaborative robotic arm developed by [F&P Robotics](https://www.fp-robotics.com).
```

File: docs/blog/Webots-2020-b-release.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+---
+
+### MJPEG Simulation Streaming
+
+The web interface of Webots now allows you to [stream](https://cyberbotics.com/doc/guide/web-streaming) a simulation as an MJPEG video stream.
+The main advantages of this technology compared to the X3D mode, is that it loads faster and provides high fidelity images.
+However, it requires higher bandwidth and server CPU/GPU power.
+
+---
```
Comment (DavidMansolino) [inline docs/blog/Webots-2020-b-release.md:112]:
```suggestion
---

### Share Your Project with the Community

We have created a new [Github repository](https://github.com/cyberbotics/community-projects) for the community to share your PROTO nodes, controllers, worlds, etc.
We hope to receive many [contributions](https://github.com/cyberbotics/community-projects/blob/master/CONTRIBUTING.md) from the community.

---
```

**Pull request merged**